llrt_core/src/modules/net/mod.rs

```diff
@@ -41,41 +50,53 @@ pub fn get_pool_idle_timeout() -> u64 {
     pool_idle_timeout
 }
 
-pub static HTTP_CLIENT: Lazy<Client<HttpsConnector<HttpConnector>, Full<Bytes>>> =
-    Lazy::new(|| {
-        let pool_idle_timeout: u64 = get_pool_idle_timeout();
+pub static HTTP_CLIENT: Lazy<
+    ClientConfigResult<Client<HttpsConnector<HttpConnector>, Full<Bytes>>>,
```

Comment:
Can we return an ioresult instead and avoid pulling in a dependency?

Reply:
Sure, I've changed this to a `std::io::Result`. However, since `std::io::Error` doesn't implement Clone, the TLS config clone for the HTTP client is a bit more verbose than I'd like.